subsys/bluetooth/controller/hci/hci.c

```diff
@@ -952,17 +974,21 @@ static int controller_cmd_handle(u8_t ocf, struct net_buf *cmd,
 		le_rem_dev_from_wl(cmd, evt);
 		break;
 
+#if defined(CONFIG_BLUETOOTH_CONTROLLER_STATE_CONN)
 	case BT_OCF(BT_HCI_OP_LE_CONN_UPDATE):
 		le_conn_update(cmd, evt);
 		break;
 
+#if defined(CONFIG_BLUETOOTH_CONTROLLER_ROLE_MASTER)
```

Comment:
Could you try to reorder the case statments so that each #if-#endif pair only appears once?